### PR TITLE
Parent PointLight to the camera, not to its parent

### PR DESCRIPTION
**Pull request by wheeland**
This will fix the ViewAll, but keep the lighting intact

### CODE REVIEW

**Comment by mkrus** [comment]
why not move it in the file rather than messing with parent?

**Comment by wheeland** [comment]
Because we need the light to be a child of the glTF camera, when it is active. So we need to re-parent it every time the active camera changes.

**Comment by wheeland** [comment]
Update: 
- also use this fix for the car-scene
- make linear and quadratic attenuation 0.0, so that the light has a constant strength, like DirectionalLight
- decrease constantAttenuation to make it brighter

File: examples/kuesa/car-scene/qml/MainScene.qml [hunk 1]
```diff
@@ -169,14 +169,12 @@ Kuesa.SceneEntity {
...
-        property Camera followedCamera: frameGraph.camera
-        parent: followedCamera.parent
+        parent: frameGraph.camera
         components: [
             PointLight {
-                constantAttenuation: 5.0    // don't illuminate too much, but keep the shininess
-            },
-            Transform {
-                translation: pointLightEntity.followedCamera.position
+                constantAttenuation: 1.0
```
Comment (mkrus) [inline examples/kuesa/car-scene/qml/MainScene.qml]: those are the default I think

**Pull request merged**